comps/third_parties/video-llama/src/Dockerfile

```
@@ -30,12 +30,16 @@ RUN pip install --no-cache-dir --upgrade pip setuptools && \
     fi
 
 ARG VIDEO_LLAMA_REPO=https://github.com/DAMO-NLP-SG/Video-LLaMA.git
-ARG VIDEO_LLAMA_COMMIT=0adb19e
-RUN git clone ${VIDEO_LLAMA_REPO} Video-LLaMA && \
-    cd Video-LLaMA && git checkout ${VIDEO_LLAMA_COMMIT} && \
+ARG VIDEO_LLAMA_COMMIT=0adb19e829254b46a442ee78cdcd609a988c5de7
+
+RUN git clone --depth=1 ${VIDEO_LLAMA_REPO} && \
+    cd /home/user/comps/third_parties/video-llama/src/Video-LLaMA && \
```

IMHO clearer with the just cloned repo name (now that the Hadolint warning about relative paths is ignored):
```suggestion
    cd Video-LLaMA && \
```